src/util/types.ts

```diff
@@ -1127,7 +1135,7 @@ export type ViewBox = CartesianViewBox | PolarViewBox;
 export const filterProps = (
```

Comment:
I wonder, would it make sense to extract this function into a separate file and add a minimal unit test - just to set a precedent and an example for future changes?

Reply:
This becomes a lot bigger refactor if I do that due to many changing imports. Do you think its acceptable to add a unit test now adjacent to the file it will be in and refactor in a different PR?